code/modules/vending/clothesmate.dm

```diff
@@ -7,46 +7,17 @@
 	product_slogans = "Dress for success!;Prepare to look swagalicious!;Look at all this swag!;Why leave style up to fate? Use the ClothesMate!"
 	vend_reply = "Thank you for using the ClothesMate!"
 	products = list(
-		/obj/item/clothing/head/beanie = 3,
-		/obj/item/clothing/head/beanie/black = 3,
-		/obj/item/clothing/head/beanie/red = 3,
-		/obj/item/clothing/head/beanie/green = 3,
-		/obj/item/clothing/head/beanie/darkblue = 3,
-		/obj/item/clothing/head/beanie/purple = 3,
-		/obj/item/clothing/head/beanie/yellow = 3,
-		/obj/item/clothing/head/beanie/orange = 3,
-		/obj/item/clothing/head/beanie/cyan = 3,
-		/obj/item/clothing/head/beanie/christmas = 3,
-		/obj/item/clothing/head/beanie/striped = 3,
-		/obj/item/clothing/head/beanie/stripedred = 3,
-		/obj/item/clothing/head/beanie/stripedblue = 3,
-		/obj/item/clothing/head/beanie/stripedgreen = 3,
-		/obj/item/clothing/head/beanie/rasta = 3,
+		/obj/item/clothing/head/wig/natural = 4,
+		/obj/item/clothing/head/beanie = 8,
 		/obj/item/clothing/head/beret/black = 3,
-		/obj/item/clothing/head/kippah = 3,
-		/obj/item/clothing/head/taqiyahred = 3,
 		/obj/item/clothing/mask/bandana = 3,
 		/obj/item/clothing/mask/bandana/striped = 3,
 		/obj/item/clothing/mask/bandana/skull = 3,
-		/obj/item/clothing/gloves/fingerless = 2,
-		/obj/item/clothing/neck/scarf/pink = 3,
-		/obj/item/clothing/neck/scarf/red = 3,
-		/obj/item/clothing/neck/scarf/green = 3,
-		/obj/item/clothing/neck/scarf/darkblue = 3,
-		/obj/item/clothing/neck/scarf/purple = 3,
-		/obj/item/clothing/neck/scarf/yellow = 3,
-		/obj/item/clothing/neck/scarf/orange = 3,
-		/obj/item/clothing/neck/scarf/cyan = 3,
-		/obj/item/clothing/neck/scarf = 3,
-		/obj/item/clothing/neck/scarf/black = 3,
-		/obj/item/clothing/neck/scarf/zebra = 3,
-		/obj/item/clothing/neck/scarf/christmas = 3,
-		/obj/item/clothing/neck/stripedredscarf = 3,
-		/obj/item/clothing/neck/stripedbluescarf = 3,
-		/obj/item/clothing/neck/stripedgreenscarf = 3,
+		/obj/item/clothing/neck/scarf = 6,
+		/obj/item/clothing/neck/large_scarf = 6,
 		/obj/item/clothing/neck/tie = 6,
-		/obj/item/clothing/neck/tie/horrible = 3,
 		/obj/item/clothing/accessory/pride = 15,
+<<<<<<< HEAD
```

Comment:
dunno whats happening here, but my review is so I can open a manual mirror anyways so I'll fix it there